Updates Read me for mono repo usage with shared configs. (#600)
lukeautry/tsoa: README.MD

# tsoa

Pronounced so·uh

[![Codeship Status for lukeautry/tsoa](https://codeship.com/projects/cdce38d0-1f6b-0134-258e-1ed679ae6c9d/status?branch=master)](https://codeship.com/projects/160322)
[![npm version](https://badge.fury.io/js/tsoa.svg)](https://badge.fury.io/js/tsoa)

## Table of Contents

> 

Reviewer: It seems like you have a formatter running on save, would you mind resubmitting this PR without the noise?

Author: @WoH I'll clear the formatting and resubmit.

  }
}
```

> Note: tsoa can not create swagger documents from interfaces that are defined in external dependencies. This is by design. Full explanation available in [ExternalInterfacesExplanation.MD](https://github.com/lukeautry/tsoa/blob/master/docs/ExternalInterfacesExplanation.MD)

Reviewer: ```suggestion
Note: tsoa can not create swagger documents from interfaces that are not part of your program defined in your tsConfig. This is by design. Full explanation available in [ExternalInterfacesExplanation.MD](https://github.com/lukeautry/tsoa/blob/master/docs/ExternalInterfacesExplanation.MD)
```

> ### Mono Repos

Reviewer: This should be part of the External Interfaces Explanation.

Author: @WoH should I update the external interfaces only or do you want information about this in the base readme as well?

Reviewer: Mostly the external interfaces, I added a small change to the base readme as a suggestion/inspiration because I think the language was not very clear.
